sdk/java/build.gradle

```diff
@@ -43,7 +43,7 @@ repositories {
 dependencies {
     implementation("com.google.code.findbugs:jsr305:3.0.2")
     implementation("com.google.code.gson:gson:2.8.9")
-    implementation("com.pulumi:aws:5.4.0")
+    implementation("com.pulumi:aws:5.31.0")
```

Comment:
Looks (from the failing action https://github.com/pulumi/pulumi-eks/actions/runs/4416938756/jobs/7742045747#step:16:40) like this is calculated and written here, rather than read from here. I would have changed it here, too.